include/quicr/detail/messages.h

```diff
@@ -369,14 +369,17 @@ namespace quicr::messages {
     struct Fetch
     {
         uint64_t subscribe_id;
-        TrackNamespace track_namespace;
-        TrackName track_name;
         ObjectPriority priority;
         GroupOrder group_order;
+        uint64_t fetch_type;
```

Comment:
We should add an enum class for this one.  It should be as as follows for draft-08. 

```c++
enum class FetchTypes: uint64_t
{
    kStandAlone = 0x1,
    kJoiningFetch
};
```